core/src/lib/internal/net/FetchTransport.ts

```diff
@@ -1,23 +1,31 @@
-import BasicRequest from "./BasicRequest";
+import { BasicRequest } from "./BasicRequest";
 import Response from "./Response";
 import Transport from "./Transport";
+require('es6-promise').polyfill();
```

Comment:
This is how you're supposed to use `fetch-everywhere` according to its docs. This seems weird when mixed with the `import`s but I wasn't sure if there's another proper way to do it in ES6.

Reply:
Oh well!